coap_format_spec.md

```diff
@@ -92,13 +92,16 @@ urther defining the resource. In this way, the URI may *roughly* mirror the
 OCF resource definition 'type' (rt). However, this URI scheme is suggested and
 not required.
 
-The following option type *should* be present when sending content:
+(There may be multiple `uri-path` options present, e.g. 'batt/main' and
```

Comment:
This is a bit misleading when reading the examples in the Appendix. It seems like when there are multiple levels to the path, that there is one `uri-path` option per level when serialized (for example, one for `batt` and one for `main`). This is what I think needs to be clear here, as well as that all the examples given already are valid `uri-path`s.

Reply:
ok, I will try to clarify here.